guardian/editions
# Add a setting to hide or show the weather

## Summary

This is the first step for the [changes regarding weather](https://trello.com/c/Ii5WqyrA/895-make-weather-in-app-use-geolocation-data-lat-long-rather-than-ip-address). This simply adds a new setting to have weather show up or be hidden.

## Test Plan

1. Open Android emulator, install app, open app.
2. Observe weather is showing up.
3. Open settings page, observe "Display Weather" selector, set to shown.
4. Toggle "Display Weather" to be off
5. Go back to home page, observe the weather is now hidden.
6. Kill the app, restart the app. Observe that the weather is still hidden.
7. Go to settings, toggle it back on. Observe it is now showing up on the front page.
8. Kill the app, restart the app. Observe that the weather is still shown.
9. Install the app in the iPad, open. Repeat steps 1 to 8, verifying the design when weather is not shown.

<div>
<img width="300" alt="Screenshot 2019-10-23 at 13 38 53" src="https://user-images.githubusercontent.com/1733570/67395249-8580eb80-f59d-11e9-9ad6-dd0a41e1fda9.png">
<img width="300" alt="Screenshot 2019-10-23 at 13 38 42" src="https://user-images.githubusercontent.com/1733570/67395250-86198200-f59d-11e9-829f-b2b34020e2fe.png">
<img width="300" alt="Screenshot 2019-10-23 at 13 38 07" src="https://user-images.githubusercontent.com/1733570/67395251-86b21880-f59d-11e9-8fcc-0508d703ddbd.png">
<img width="300" alt="Screenshot 2019-10-23 at 13 37 59" src="https://user-images.githubusercontent.com/1733570/67395252-86b21880-f59d-11e9-8870-6d4eb8896276.png">
</div>
